### ISSUE TITLE
How to set root user/password in cluster?

### ISSUE THREAD

**Issue by Rellopn** [bitnami/bitnami-docker-etcd#26]
**Description**

**Steps to reproduce the issue:**
I have a etcd cluster.I want to add root user/password in this cluster.`ETCD_ROOT_PASSWORD`  are set in the `docker-compose.yaml`, but they don’t seem to take effect.

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version**

- Output of `docker version`:

```
Version:          20.10.8
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**
Here is my `docker-compose.yaml`:
```yaml
#.env
#ETCD_TAG=3.5.1

version: "3.5"
networks:
  base_service:
    driver: bridge
    name: project_network
etcd1:
    image: 'bitnami/etcd:${ETCD_TAG}'
    deploy:
      replicas: 1
      restart_policy:
        condition: on-failure
    volumes:
      - "./etcd/d/d1:/bitnami/etcd/"
    environment:
      - ETCD_ROOT_PASSWORD=Ja78lX222VPLn8ibs
      - ETCD_NAME=etcd1
      - ETCD_INITIAL_ADVERTISE_PEER_URLS=http://etcd1:2380
      - ETCD_LISTEN_PEER_URLS=http://0.0.0.0:2380
      - ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379
      - ETCD_ADVERTISE_CLIENT_URLS=http://etcd1:2379
      - ETCD_INITIAL_CLUSTER_TOKEN=etcd-cluster
      - ETCD_INITIAL_CLUSTER=etcd1=http://etcd1:2380,etcd2=http://etcd2:2380,etcd3=http://etcd3:2380
      - ETCD_INITIAL_CLUSTER_STATE=new
      - ETCD_DATA_DIR=/opt/bitnami/etcd/data
    ports:
      - 2379:2379
      - 2380:2380
    networks:
      - base_service

  etcd2:
    image: 'bitnami/etcd:${ETCD_TAG}'
    deploy:
      replicas: 1
      restart_policy:
        condition: on-failure
    volumes:
      - "./etcd/d/d2:/bitnami/etcd/"
    environment:
      - ALLOW_NONE_AUTHENTICATION=yes
      - ETCD_NAME=etcd2
      - ETCD_INITIAL_ADVERTISE_PEER_URLS=http://etcd2:2380
      - ETCD_LISTEN_PEER_URLS=http://0.0.0.0:2380
      - ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379
      - ETCD_ADVERTISE_CLIENT_URLS=http://etcd2:2379
      - ETCD_INITIAL_CLUSTER_TOKEN=etcd-cluster
      - ETCD_INITIAL_CLUSTER=etcd1=http://etcd1:2380,etcd2=http://etcd2:2380,etcd3=http://etcd3:2380
      - ETCD_INITIAL_CLUSTER_STATE=new
      - "ETCD_DATA_DIR=/opt/bitnami/etcd/data"
    ports:
      - 2389:2379
      - 2381:2380
    networks:
      - base_service

  etcd3:
    image: 'bitnami/etcd:${ETCD_TAG}'
    deploy:
      replicas: 1
      restart_policy:
        condition: on-failure
    volumes:
      - "./etcd/d/d3:/bitnami/etcd/"
    environment:
      - ALLOW_NONE_AUTHENTICATION=yes
      - ETCD_NAME=etcd3
      - ETCD_INITIAL_ADVERTISE_PEER_URLS=http://etcd3:2380
      - ETCD_LISTEN_PEER_URLS=http://0.0.0.0:2380
      - ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379
      - ETCD_ADVERTISE_CLIENT_URLS=http://etcd3:2379
      - ETCD_INITIAL_CLUSTER_TOKEN=etcd-cluster
      - ETCD_INITIAL_CLUSTER=etcd1=http://etcd1:2380,etcd2=http://etcd2:2380,etcd3=http://etcd3:2380
      - ETCD_INITIAL_CLUSTER_STATE=new
      - "ETCD_DATA_DIR=/opt/bitnami/etcd/data"
    ports:
      - 2399:2379
      - 2382:2380
    networks:
      - base_service
```

**Comment by rafariossaa** [comment]
Hi,
  Did you get any error ? Could you enable debug `BITNAMI_DEBUG=1` and check the logs ?

**Comment by Rellopn** [comment]
I have modified the yaml file add ```BITNAMI_DEBUG=1```,then I got an error:
```
etcd 01:23:43.09 

etcd 01:23:43.09 Welcome to the Bitnami etcd container

etcd 01:23:43.10 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-etcd

etcd 01:23:43.10 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-etcd/issues

etcd 01:23:43.12 

etcd 01:23:43.12 INFO  ==> ** Starting etcd setup **

etcd 01:23:43.18 INFO  ==> Validating settings in ETCD_* env vars..

etcd 01:23:43.18 INFO  ==> Initializing etcd

etcd 01:23:43.21 INFO  ==> There is no data from previous deployments

etcd 01:23:43.21 INFO  ==> Bootstrapping a new cluster

etcd 01:23:43.22 INFO  ==> Enabling etcd authentication

etcd 01:23:43.24 INFO  ==> Starting etcd in background

/opt/bitnami/scripts/libos.sh: line 336: 1: command not found
```
The ```BITNAMI_DEBUG``` variable seem is a boolean value.
So I set ```BITNAMI_DEBUG=true```, and this is the information output from the console.
```BITNAMI_DEBUG=true``` It doesn't seem to work.
here is  modified```docker-compose.yaml``` etcd1:
```
  etcd1:
    image: 'bitnami/etcd:${ETCD_TAG}'
    deploy:
      replicas: 1
      restart_policy:
        condition: on-failure
    volumes:
      - "./etcd/d/d1:/bitnami/etcd/"
    environment:
      - BITNAMI_DEBUG=true
      - ETCD_ROOT_PASSWORD=Ja4vXCP2VPLn8ibs
      - ETCD_NAME=etcd1
      - ETCD_INITIAL_ADVERTISE_PEER_URLS=http://etcd1:2380
      - ETCD_LISTEN_PEER_URLS=http://0.0.0.0:2380
      - ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379
      - ETCD_ADVERTISE_CLIENT_URLS=http://etcd1:2379
      - ETCD_INITIAL_CLUSTER_TOKEN=etcd-cluster
      - ETCD_INITIAL_CLUSTER=etcd1=http://etcd1:2380,etcd2=http://etcd2:2380,etcd3=http://etcd3:2380
      - ETCD_INITIAL_CLUSTER_STATE=new
      - ETCD_DATA_DIR=/opt/bitnami/etcd/data
    ports:
      - 2379:2379
      - 2380:2380
    networks:
      - base_service
```

console info:

```
etcd 01:04:00.65 

etcd 01:04:00.65 Welcome to the Bitnami etcd container

etcd 01:04:00.65 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-etcd

etcd 01:04:00.66 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-etcd/issues

etcd 01:04:00.66 

etcd 01:04:00.66 INFO  ==> ** Starting etcd setup **

etcd 01:04:00.69 INFO  ==> Validating settings in ETCD_* env vars..

etcd 01:04:00.70 INFO  ==> Initializing etcd

etcd 01:04:00.72 INFO  ==> Detected data from previous deployments

etcd 01:04:00.75 INFO  ==> Member ID wasn't properly stored, the member will try to join the cluster by it's own

etcd 01:04:00.75 INFO  ==> ** etcd setup finished! **

etcd 01:04:00.78 INFO  ==> ** Starting etcd **

{"level":"info","ts":1638234240.8186646,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_ADVERTISE_CLIENT_URLS","variable-value":"http://etcd1:2379"}

{"level":"info","ts":1638234240.8187392,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_AUTO_TLS","variable-value":"false"}

{"level":"info","ts":1638234240.8187563,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_CLIENT_CERT_AUTH","variable-value":"false"}

{"level":"info","ts":1638234240.8187673,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_DATA_DIR","variable-value":"/bitnami/etcd/data"}

{"level":"info","ts":1638234240.8189852,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_INITIAL_ADVERTISE_PEER_URLS","variable-value":"http://etcd1:2380"}

{"level":"info","ts":1638234240.8190289,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_INITIAL_CLUSTER","variable-value":"etcd1=http://etcd1:2380,etcd2=http://etcd2:2380,etcd3=http://etcd3:2380"}

{"level":"info","ts":1638234240.819082,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_INITIAL_CLUSTER_STATE","variable-value":"new"}

{"level":"info","ts":1638234240.8192055,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_INITIAL_CLUSTER_TOKEN","variable-value":"etcd-cluster"}

{"level":"info","ts":1638234240.8192332,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_LISTEN_CLIENT_URLS","variable-value":"http://0.0.0.0:2379"}

{"level":"info","ts":1638234240.8192942,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_LISTEN_PEER_URLS","variable-value":"http://0.0.0.0:2380"}

{"level":"info","ts":1638234240.819309,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_LOG_LEVEL","variable-value":"info"}

{"level":"info","ts":1638234240.8193243,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_NAME","variable-value":"etcd1"}


{"level":"info","ts":1638234240.8193324,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_PEER_AUTO_TLS","variable-value":"false"}

{"level":"warn","ts":1638234240.819462,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_TRUSTED_CA_FILE="}

{"level":"warn","ts":1638234240.8194988,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_ON_K8S=no"}

{"level":"warn","ts":1638234240.8195114,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_SNAPSHOTS_DIR=/snapshots"}

{"level":"warn","ts":1638234240.8195188,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_BIN_DIR=/opt/bitnami/etcd/sbin"}

{"level":"warn","ts":1638234240.8195267,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_VOLUME_DIR=/bitnami/etcd"}

{"level":"warn","ts":1638234240.8195422,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_CLUSTER_DOMAIN="}

{"level":"warn","ts":1638234240.819722,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_DISASTER_RECOVERY=no"}

{"level":"warn","ts":1638234240.8197367,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_KEY_FILE="}

{"level":"warn","ts":1638234240.819745,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_DAEMON_GROUP=etcd"}

{"level":"warn","ts":1638234240.8200336,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_START_FROM_SNAPSHOT=no"}

{"level":"warn","ts":1638234240.8200488,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_INIT_SNAPSHOT_FILENAME="}

{"level":"warn","ts":1638234240.820062,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_INIT_SNAPSHOTS_DIR=/init-snapshot"}

{"level":"warn","ts":1638234240.8200686,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_BASE_DIR=/opt/bitnami/etcd"}

{"level":"warn","ts":1638234240.8200781,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_CERT_FILE="}

{"level":"warn","ts":1638234240.8200924,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_NEW_MEMBERS_ENV_FILE=/bitnami/etcd/data/new_member_envs"}

{"level":"warn","ts":1638234240.8202684,"caller":"flags/flag.go:93","msg":"unrecognized environment variable","environment-variable":"ETCD_DAEMON_USER=etcd"}

{"level":"info","ts":"2021-11-30T01:04:00.820Z","caller":"etcdmain/etcd.go:72","msg":"Running: ","args":["/opt/bitnami/etcd/bin/etcd"]}

{"level":"info","ts":"2021-11-30T01:04:00.820Z","caller":"etcdmain/etcd.go:115","msg":"server has been already initialized","data-dir":"/bitnami/etcd/data","dir-type":"member"}

{"level":"info","ts":"2021-11-30T01:04:00.820Z","caller":"embed/etcd.go:131","msg":"configuring peer listeners","listen-peer-urls":["http://0.0.0.0:2380"]}

{"level":"info","ts":"2021-11-30T01:04:00.821Z","caller":"embed/etcd.go:139","msg":"configuring client listeners","listen-client-urls":["http://0.0.0.0:2379"]}

{"level":"info","ts":"2021-11-30T01:04:00.823Z","caller":"embed/etcd.go:307","msg":"starting an etcd server","etcd-version":"3.5.1","git-sha":"e8732fb5f","go-version":"go1.16.3","go-os":"linux","go-arch":"amd64","max-cpu-set":2,"max-cpu-available":2,"member-initialized":true,"name":"etcd1","data-dir":"/bitnami/etcd/data","wal-dir":"","wal-dir-dedicated":"","member-dir":"/bitnami/etcd/data/member","force-new-cluster":false,"heartbeat-interval":"100ms","election-timeout":"1s","initial-election-tick-advance":true,"snapshot-count":100000,"snapshot-catchup-entries":5000,"initial-advertise-peer-urls":["http://etcd1:2380"],"listen-peer-urls":["http://0.0.0.0:2380"],"advertise-client-urls":["http://etcd1:2379"],"listen-client-urls":["http://0.0.0.0:2379"],"listen-metrics-urls":[],"cors":["*"],"host-whitelist":["*"],"initial-cluster":"","initial-cluster-state":"new","initial-cluster-token":"","quota-size-bytes":2147483648,"pre-vote":true,"initial-corrupt-check":false,"corrupt-check-time-interval":"0s","auto-compaction-mode":"periodic","auto-compaction-retention":"0s","auto-compaction-interval":"0s","discovery-url":"","discovery-proxy":"","downgrade-check-interval":"5s"}

{"level":"info","ts":"2021-11-30T01:04:00.839Z","caller":"etcdserver/backend.go:81","msg":"opened backend db","path":"/bitnami/etcd/data/member/snap/db","took":"4.509353ms"}

{"level":"info","ts":"2021-11-30T01:04:00.854Z","caller":"etcdserver/server.go:526","msg":"No snapshot found. Recovering WAL from scratch!"}

{"level":"info","ts":"2021-11-30T01:04:00.861Z","caller":"etcdserver/raft.go:483","msg":"restarting local member","cluster-id":"8f2a2ec5c0087dcb","local-member-id":"ade526d28b1f92f7","commit-index":10}

{"level":"info","ts":"2021-11-30T01:04:00.861Z","logger":"raft","caller":"etcdserver/zap_raft.go:77","msg":"ade526d28b1f92f7 switched to configuration voters=()"}

{"level":"info","ts":"2021-11-30T01:04:00.861Z","logger":"raft","caller":"etcdserver/zap_raft.go:77","msg":"ade526d28b1f92f7 became follower at term 2"}

{"level":"info","ts":"2021-11-30T01:04:00.861Z","logger":"raft","caller":"etcdserver/zap_raft.go:77","msg":"newRaft ade526d28b1f92f7 [peers: [], term: 2, commit: 10, applied: 0, lastindex: 10, lastterm: 2]"}

{"level":"warn","ts":"2021-11-30T01:04:00.866Z","caller":"auth/store.go:1220","msg":"simple token is not cryptographically signed"}

{"level":"info","ts":"2021-11-30T01:04:00.869Z","caller":"mvcc/kvstore.go:415","msg":"kvstore restored","current-rev":2}

{"level":"info","ts":"2021-11-30T01:04:00.871Z","caller":"etcdserver/quota.go:94","msg":"enabled backend quota with default value","quota-name":"v3-applier","quota-size-bytes":2147483648,"quota-size":"2.1 GB"}

{"level":"info","ts":"2021-11-30T01:04:00.873Z","caller":"etcdserver/server.go:843","msg":"starting etcd server","local-member-id":"ade526d28b1f92f7","local-server-version":"3.5.1","cluster-version":"to_be_decided"}

{"level":"info","ts":"2021-11-30T01:04:00.875Z","caller":"etcdserver/server.go:744","msg":"starting initial election tick advance","election-ticks":10}

{"level":"info","ts":"2021-11-30T01:04:00.875Z","logger":"raft","caller":"etcdserver/zap_raft.go:77","msg":"ade526d28b1f92f7 switched to configuration voters=(12530464223947363063)"}

{"level":"info","ts":"2021-11-30T01:04:00.875Z","caller":"membership/cluster.go:421","msg":"added member","cluster-id":"8f2a2ec5c0087dcb","local-member-id":"ade526d28b1f92f7","added-peer-id":"ade526d28b1f92f7","added-peer-peer-urls":["http://etcd1:2380"]}

{"level":"info","ts":"2021-11-30T01:04:00.875Z","logger":"raft","caller":"etcdserver/zap_raft.go:77","msg":"ade526d28b1f92f7 switched to configuration voters=(12530464223947363063 13634804518189357139)"}

{"level":"info","ts":"2021-11-30T01:04:00.876Z","caller":"membership/cluster.go:421","msg":"added member","cluster-id":"8f2a2ec5c0087dcb","local-member-id":"ade526d28b1f92f7","added-peer-id":"bd388e7810915853","added-peer-peer-urls":["http://etcd3:2380"]}

{"level":"info","ts":"2021-11-30T01:04:00.876Z","caller":"rafthttp/peer.go:133","msg":"starting remote peer","remote-peer-id":"bd388e7810915853"}

{"level":"info","ts":"2021-11-30T01:04:00.876Z","caller":"rafthttp/pipeline.go:72","msg":"started HTTP pipelining with remote peer","local-member-id":"ade526d28b1f92f7","remote-peer-id":"bd388e7810915853"}

{"level":"info","ts":"2021-11-30T01:04:00.877Z","caller":"rafthttp/stream.go:169","msg":"started stream writer with remote peer","local-member-id":"ade526d28b1f92f7","remote-peer-id":"bd388e7810915853"}

{"level":"info","ts":"2021-11-30T01:04:00.878Z","caller":"rafthttp/stream.go:169","msg":"started stream writer with remote peer","local-member-id":"ade526d28b1f92f7","remote-peer-id":"bd388e7810915853"}

{"level":"info","ts":"2021-11-30T01:04:00.878Z","caller":"rafthttp/peer.go:137","msg":"started remote peer","remote-peer-id":"bd388e7810915853"}

{"level":"info","ts":"2021-11-30T01:04:00.878Z","caller":"rafthttp/transport.go:317","msg":"added remote peer","local-member-id":"ade526d28b1f92f7","remote-peer-id":"bd388e7810915853","remote-peer-urls":["http://etcd3:2380"]}

{"level":"info","ts":"2021-11-30T01:04:00.878Z","logger":"raft","caller":"etcdserver/zap_raft.go:77","msg":"ade526d28b1f92f7 switched to configuration voters=(12530464223947363063 13634804518189357139 15168875803774599630)"}

{"level":"info","ts":"2021-11-30T01:04:00.878Z","caller":"membership/cluster.go:421","msg":"added member","cluster-id":"8f2a2ec5c0087dcb","local-member-id":"ade526d28b1f92f7","added-peer-id":"d282ac2ce600c1ce","added-peer-peer-urls":["http://etcd2:2380"]}

{"level":"info","ts":"2021-11-30T01:04:00.878Z","caller":"rafthttp/peer.go:133","msg":"starting remote peer","remote-peer-id":"d282ac2ce600c1ce"}

{"level":"info","ts":"2021-11-30T01:04:00.879Z","caller":"rafthttp/pipeline.go:72","msg":"started HTTP pipelining with remote peer","local-member-id":"ade526d28b1f92f7","remote-peer-id":"d282ac2ce600c1ce"}

{"level":"info","ts":"2021-11-30T01:04:00.880Z","caller":"embed/etcd.go:276","msg":"now serving peer/client/metrics","local-member-id":"ade526d28b1f92f7","initial-advertise-peer-urls":["http://etcd1:2380"],"listen-peer-urls":["http://0.0.0.0:2380"],"advertise-client-urls":["http://etcd1:2379"],"listen-client-urls":["http://0.0.0.0:2379"],"listen-metrics-urls":[]}

{"level":"info","ts":"2021-11-30T01:04:00.881Z","caller":"embed/etcd.go:580","msg":"serving peer traffic","address":"[::]:2380"}

{"level":"info","ts":"2021-11-30T01:04:00.881Z","caller":"embed/etcd.go:552","msg":"cmux::serve","address":"[::]:2380"}

{"level":"info","ts":"2021-11-30T01:04:00.881Z","caller":"rafthttp/stream.go:395","msg":"started stream reader with remote peer","stream-reader-type":"stream MsgApp v2","local-member-id":"ade526d28b1f92f7","remote-peer-id":"bd388e7810915853"}

{"level":"info","ts":"2021-11-30T01:04:00.882Z","caller":"rafthttp/stream.go:395","msg":"started stream reader with remote peer","stream-reader-type":"stream Message","local-member-id":"ade526d28b1f92f7","remote-peer-id":"bd388e7810915853"}

{"level":"info","ts":"2021-11-30T01:04:00.886Z","caller":"rafthttp/peer_status.go:53","msg":"peer became active","peer-id":"bd388e7810915853"}

{"level":"info","ts":"2021-11-30T01:04:00.886Z","caller":"rafthttp/stream.go:412","msg":"established TCP streaming connection with remote peer","stream-reader-type":"stream MsgApp v2","local-member-id":"ade526d28b1f92f7","remote-peer-id":"bd388e7810915853"}

{"level":"info","ts":"2021-11-30T01:04:00.887Z","caller":"rafthttp/stream.go:412","msg":"established TCP streaming connection with remote peer","stream-reader-type":"stream Message","local-member-id":"ade526d28b1f92f7","remote-peer-id":"bd388e7810915853"}

{"level":"info","ts":"2021-11-30T01:04:00.891Z","caller":"rafthttp/stream.go:169","msg":"started stream writer with remote peer","local-member-id":"ade526d28b1f92f7","remote-peer-id":"d282ac2ce600c1ce"}

{"level":"info","ts":"2021-11-30T01:04:00.892Z","caller":"rafthttp/stream.go:169","msg":"started stream writer with remote peer","local-member-id":"ade526d28b1f92f7","remote-peer-id":"d282ac2ce600c1ce"}

{"level":"info","ts":"2021-11-30T01:04:00.892Z","caller":"rafthttp/peer.go:137","msg":"started remote peer","remote-peer-id":"d282ac2ce600c1ce"}

{"level":"info","ts":"2021-11-30T01:04:00.892Z","caller":"rafthttp/transport.go:317","msg":"added remote peer","local-member-id":"ade526d28b1f92f7","remote-peer-id":"d282ac2ce600c1ce","remote-peer-urls":["http://etcd2:2380"]}

{"level":"info","ts":"2021-11-30T01:04:00.892Z","caller":"rafthttp/stream.go:395","msg":"started stream reader with remote peer","stream-reader-type":"stream Message","local-member-id":"ade526d28b1f92f7","remote-peer-id":"d282ac2ce600c1ce"}

{"level":"info","ts":"2021-11-30T01:04:00.892Z","caller":"membership/cluster.go:584","msg":"set initial cluster version","cluster-id":"8f2a2ec5c0087dcb","local-member-id":"ade526d28b1f92f7","cluster-version":"3.5"}

{"level":"info","ts":"2021-11-30T01:04:00.892Z","caller":"api/capability.go:75","msg":"enabled capabilities for version","cluster-version":"3.5"}

{"level":"info","ts":"2021-11-30T01:04:00.893Z","caller":"rafthttp/stream.go:395","msg":"started stream reader with remote peer","stream-reader-type":"stream MsgApp v2","local-member-id":"ade526d28b1f92f7","remote-peer-id":"d282ac2ce600c1ce"}

{"level":"info","ts":"2021-11-30T01:04:00.894Z","caller":"rafthttp/peer_status.go:53","msg":"peer became active","peer-id":"d282ac2ce600c1ce"}

{"level":"info","ts":"2021-11-30T01:04:00.894Z","caller":"rafthttp/stream.go:412","msg":"established TCP streaming connection with remote peer","stream-reader-type":"stream Message","local-member-id":"ade526d28b1f92f7","remote-peer-id":"d282ac2ce600c1ce"}

{"level":"info","ts":"2021-11-30T01:04:00.902Z","caller":"rafthttp/stream.go:412","msg":"established TCP streaming connection with remote peer","stream-reader-type":"stream MsgApp v2","local-member-id":"ade526d28b1f92f7","remote-peer-id":"d282ac2ce600c1ce"}

{"level":"info","ts":"2021-11-30T01:04:00.925Z","caller":"etcdserver/server.go:767","msg":"initialized peer connections; fast-forwarding election ticks","local-member-id":"ade526d28b1f92f7","forward-ticks":8,"forward-duration":"800ms","election-ticks":10,"election-timeout":"1s","active-remote-members":2}

{"level":"info","ts":"2021-11-30T01:04:00.940Z","logger":"raft","caller":"etcdserver/zap_raft.go:77","msg":"raft.node: ade526d28b1f92f7 elected leader d282ac2ce600c1ce at term 2"}

{"level":"info","ts":"2021-11-30T01:04:00.949Z","caller":"etcdserver/server.go:2027","msg":"published local member to cluster through raft","local-member-id":"ade526d28b1f92f7","local-member-attributes":"{Name:etcd1 ClientURLs:[http://etcd1:2379]}","request-path":"/0/members/ade526d28b1f92f7/attributes","cluster-id":"8f2a2ec5c0087dcb","publish-timeout":"7s"}

{"level":"info","ts":"2021-11-30T01:04:00.950Z","caller":"embed/serve.go:98","msg":"ready to serve client requests"}

{"level":"info","ts":"2021-11-30T01:04:00.951Z","caller":"embed/serve.go:140","msg":"serving client traffic insecurely; this is strongly discouraged!","address":"[::]:2379"}

{"level":"info","ts":"2021-11-30T01:04:00.952Z","caller":"etcdmain/main.go:47","msg":"notifying init daemon"}

{"level":"info","ts":"2021-11-30T01:04:00.953Z","caller":"etcdmain/main.go:53","msg":"successfully notified init daemon"}

{"level":"info","ts":"2021-11-30T01:04:01.363Z","caller":"rafthttp/stream.go:249","msg":"set message encoder","from":"ade526d28b1f92f7","to":"d282ac2ce600c1ce","stream-type":"stream MsgApp v2"}

{"level":"info","ts":"2021-11-30T01:04:01.363Z","caller":"rafthttp/stream.go:274","msg":"established TCP streaming connection with remote peer","stream-writer-type":"stream MsgApp v2","local-member-id":"ade526d28b1f92f7","remote-peer-id":"d282ac2ce600c1ce"}

{"level":"info","ts":"2021-11-30T01:04:01.363Z","caller":"rafthttp/stream.go:249","msg":"set message encoder","from":"ade526d28b1f92f7","to":"bd388e7810915853","stream-type":"stream MsgApp v2"}

{"level":"info","ts":"2021-11-30T01:04:01.363Z","caller":"rafthttp/stream.go:274","msg":"established TCP streaming connection with remote peer","stream-writer-type":"stream MsgApp v2","local-member-id":"ade526d28b1f92f7","remote-peer-id":"bd388e7810915853"}

{"level":"info","ts":"2021-11-30T01:04:01.364Z","caller":"rafthttp/stream.go:249","msg":"set message encoder","from":"ade526d28b1f92f7","to":"d282ac2ce600c1ce","stream-type":"stream Message"}

{"level":"info","ts":"2021-11-30T01:04:01.364Z","caller":"rafthttp/stream.go:274","msg":"established TCP streaming connection with remote peer","stream-writer-type":"stream Message","local-member-id":"ade526d28b1f92f7","remote-peer-id":"d282ac2ce600c1ce"}

{"level":"info","ts":"2021-11-30T01:04:01.369Z","caller":"rafthttp/stream.go:249","msg":"set message encoder","from":"ade526d28b1f92f7","to":"bd388e7810915853","stream-type":"stream Message"}

{"level":"info","ts":"2021-11-30T01:04:01.370Z","caller":"rafthttp/stream.go:274","msg":"established TCP streaming connection with remote peer","stream-writer-type":"stream Message","local-member-id":"ade526d28b1f92f7","remote-peer-id":"bd388e7810915853"}
```
look forward to your help.

**Comment by Rellopn** [comment]
This problem seems to have been solved in v3.5.2

**Comment by carrodher** [comment]
Thanks for confirming! I'm closing the issue, please, feel free to reopen it or create another one if you find any issue
